app/assets/stylesheets/active_admin/components/_tables.scss

```diff
@@ -91,18 +99,28 @@ table.index_table {
   th {
     @include no-shadow;
     @include no-gradient;
-    width: 150px;
+    max-width: 150px;
     font-size: 0.9em;
-    padding-left: 0;
-    text-transform: uppercase;
     color: $primary-color;
     @include text-shadow;
   }
+  thead th {
+  }
+  tbody th {
```

Comment:
Empty rule<br>Selector should have depth of applicability no greater than 2, but was 4